### ISSUE TITLE
Electrified Fence

### ISSUE THREAD

**Issue by LokyLive** [OpenModularTurretsTeam/OpenModularPassiveDefense#5]
Electrified fence consumming RF could be awesome

**Comment by Keridos** [comment]
Yeah, not entirely sure how we would implement that but It is something we plan on adding :)

**Comment by hanakocz** [comment]
Just please no RF, that system is dead and not really electricity. If so, use EU or similar electricity...

**Comment by Keridos** [comment]
The system would pretty much use  (as all OMLib based electricity) use RF, Tesla, EU and Forge Energy.

To be honest, I did not yet see a single electricity system that actually acts like real world electricity and based on my experience of that (electrodynamics) I am fairly certain that the majority of the minecraft players do not want it to be like in real life. Because with that you could argue that things like energy storage should be nerfed until they are absolutely infeasable. And loss on wires is also a big thing there and transformers and different cable and insulators, much too complex tbh.
